frontend/src/components/VMediaInfo/VSourceProviderTooltip.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <dt>
+    {{ $t(datum.label) }}
+    <VTooltip placement="top" :describedby="describedby" class="ms-1">
+      <template #default>
+        <p
+          class="caption-regular rounded-sm bg-dark-charcoal px-2 py-1 text-white"
+        >
+          {{ description }}
+        </p>
+      </template>
+    </VTooltip>
+  </dt>
+</template>
+<script lang="ts">
+import { computed, defineComponent, PropType } from "vue"
+
+import { Metadata } from "~/types/media"
+import { useI18n } from "~/composables/use-i18n"
+
+import VTooltip from "~/components/VTooltip/VTooltip.vue"
+
+export default defineComponent({
+  name: "VSourceProviderTooltip",
+  components: { VTooltip },
+  props: {
+    datum: {
+      type: Object as PropType<Metadata>,
+      required: true,
+    },
+    describedby: {
```

Comment:
Nit: it's bugging me that this does not follow camel case for the 'B' as in `describedBy`.